mono/metadata/gc.c

```diff
@@ -681,6 +681,30 @@ ves_icall_System_GCHandle_GetTarget (guint32 handle)
 	return mono_gchandle_get_target (handle);
 }
 
+static gboolean is_object_pinnable(MonoObject* obj)
```

Comment:
Declarations should go like:

    static gboolean
    is_object_pinnable (MonoObject *obj)